lib/loadMtl.js

```diff
@@ -222,6 +222,11 @@ function loadMaterialTexture(material, name, texturePath, textureOptions, mtlDir
             texturePromise = Promise.resolve();
         } else {
             texturePromise = loadTexture(texturePath, textureOptions)
+                .catch(function() {
```

Comment:
Should this also happen if `options.secure && outsideDirectory`?

Also not sure this should happen as a generic `catch`, since `loadTexture` can also reject due to a `pngjs` or `jpegjs` error. So technically this could cause the wrong texture to be loaded if the intended texture fails due to decode-related problems, although that'd probably be a pretty contrived case.

Reply:
> Should this also happen if options.secure && outsideDirectory?

Yes definitely. I pushed a fix for that.

> Also not sure this should happen as a generic catch, since loadTexture can also reject due to a pngjs or jpegjs error.

I also considered this but didn't think it would ever really happen.